resources/data/nrf_connect_sdk-0.1.3.json

```diff
@@ -49,6 +49,62 @@
                 }
             ]
         },
+        {
+            "type": "Checkable",
+            "steps": [
+                {
+                    "type": "Step",
+                    "platforms": [
+                        { "platform": "darwin" }
```

Comment:
If the steps are the same for both macOS and Linux, then you can simplify them into a single blob if you extend the _platforms_ array: `"platforms": [ { "platform": "darwin" }, { "platform": "linux" } ]`

Reply:
Yes, but steps in that case are not exactly the same, because Linux and macOS need different GN archive, so the hyperlink passed to `wget` is different for both platforms.